Sources/MapboxNavigation/LaneView.swift

```diff
@@ -36,6 +40,14 @@ extension LaneIndication {
             primaryIndication = .sharpLeft
         } else if indications.contains(.sharpRight) && maneuverDirection ?? .sharpRight == .sharpRight {
             primaryIndication = .sharpRight
+        } else if indications.contains(.left) && !indications.contains(.sharpLeft) && !indications.contains(.uTurn) && maneuverDirection ?? .sharpLeft == .sharpLeft {
+            primaryIndication = .left
+        } else if indications.contains(.right) && !indications.contains(.sharpRight) && !indications.contains(.uTurn) && maneuverDirection ?? .sharpRight == .sharpRight {
+            primaryIndication = .right
+        } else if !indications.isDisjoint(with: .lefts) && !indications.isDisjoint(with: .rights) && maneuverDirection?.isLeft ?? false {
+            primaryIndication = .left
+        } else if !indications.isDisjoint(with: .lefts) && !indications.isDisjoint(with: .rights) && maneuverDirection?.isRight ?? false {
+            primaryIndication = .right
```

Comment:
These cases handle lanes with opposite turns, such as left-right. Previously, we’d fall into the big else case below that tries to choose the “least extreme” indication. It’d be a tie between left and right, so that logic would choose left because it happens to come first. This more generic logic accounts for slight and sharp turns too.